lib/launcher/launcher.go

```diff
@@ -347,6 +348,17 @@ func (l *Launcher) PID() int {
 	return l.pid
 }
 
+// Cleanup wait until the Browser exits and release related resources
+func (l *Launcher) Cleanup() {
+	<-l.exit
+	if _, has := l.Get("keep-user-data-dir"); !has {
+		dir, _ := l.Get("user-data-dir")
+		fmt.Println(utils.C("Remove", "cyan"), dir)
```

Comment:
should use configurable log like other parts, search `l.log`:

https://github.com/go-rod/rod/blob/44b9c72/lib/launcher/launcher.go#L360